styles/globals.css

```diff
@@ -15,6 +15,21 @@ a {
   box-sizing: border-box;
 }
 
+.vertical-timeline::before {
+  background: rgb(75, 69, 69) !important;
+}
+
+.vertical-timeline-element-content {
+  box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px !important;
+}
+
+@media only screen and (max-width: 1300px) {
+  .vertical-timeline-element-date {
+    display: block !important;
+    float: none !important;
+  }
+}
```

Comment:
these should probably only be used in the relevant components rather than globally

Reply:
@eddiejaoude, this is for the styling of pseudo before selector which is present by default in the timeline component so styling within the element is not possible as they are hidden.

Reply:
@eddiejaoude, the date is given as a property in the timeline component so styling date is not possible in tailwind.

Reply:
I think the property can be styled in the timeline component?

Reply:
Hey @eddiejaoude, I try some possible cases like if I apply a border to the **VerticleTimelineElement** component then it applies to the entire section and looks like this image, 

<img width="1300" alt="Screenshot 2022-11-20 at 10 33 15 PM" src="https://user-images.githubusercontent.com/85363195/202915358-5fdeda45-8539-40ab-8675-87bf486b79ae.png">

Also if I wrap the component items inside a div and apply a border to that div then it looks something like this,

<img width="1213" alt="Screenshot 2022-11-20 at 10 38 43 PM" src="https://user-images.githubusercontent.com/85363195/202915624-8b1d6e18-871e-4b28-8f2b-102e7ac669f5.png">

The **date** is set as a property of the timeline so it is not possible to style its container within the component, same is for the pseudo **before** element.